config.php.example

```diff
@@ -81,3 +81,8 @@ Config::append('system.rest_allow',array(
     "User",
     "Contact"
 ));
+
+if (file_exists("config.override.php")) {
```

Comment:
If you want the override to occur all the time then this may not work as we cache the config.php result... in saying that though it may also just end up caching the included file the first time. If it hasn't yet been done, please test to verify